src/test/java/com/project/parkinglot/service/parking_area/impl/ParkingAreaUpdateServiceImplTest.java

```diff
@@ -0,0 +1,138 @@
+                .thenReturn(Optional.of(mockParkingAreaEntity));
+
+        // Then
+        final ParkingArea mockParkingArea =
+                parkingAreaUpdateService.parkingAreaUpdateByCapacity(mockParkingAreaId,mockParkingAreaUpdateRequest);
+
+        Assertions.assertNotNull(mockParkingArea);
+
+        Assertions.assertEquals(
+                mockParkingAreaUpdateRequest.getCapacity(),
+                mockParkingArea.getCapacity()
+        );
+
+        Assertions.assertEquals(
+                mockBeforeUpdatedParkingArea.getId(),
+                mockParkingArea.getId()
+        );
+
+        Assertions.assertEquals(
+                mockBeforeUpdatedParkingArea.getId(),
+                mockParkingArea.getId()
+        );
+
+        Assertions.assertEquals(
+                mockBeforeUpdatedParkingArea.getLocation(),
+                mockParkingArea.getLocation()
+        );
+
+        Assertions.assertEquals(
+                mockBeforeUpdatedParkingArea.getCity(),
+                mockParkingArea.getCity()
+        );
+
+        // Verify
+        Mockito.verify(parkingAreaRepository,Mockito.times(1)).findById(mockParkingAreaId);
+        Mockito.verify(parkingAreaRepository,Mockito.times(1)).save(mockParkingAreaEntity);
+
+    }
+
+    @Test
+    void givenEmptyParkingAreaUpdate_whenUpdateParkingArea_thenThrowParkingAreaNotFoundException(){
+
+        // Given
+        final ParkingAreaUpdateRequest mockParkingAreaUpdateRequest = new ParkingAreaUpdateRequestBuilder()
+                .withValidField()
+                .build();
+
+        final String mockParkingAreaUpdateRequestId = UUID.randomUUID().toString();
+
+        // When
+        Mockito.when(parkingAreaRepository.findById(mockParkingAreaUpdateRequestId)).thenReturn(Optional.empty());
+
+        // Then
+        Assertions.assertThrows(ParkingAreaNotFoundException.class,
+                () -> parkingAreaUpdateService.parkingAreaUpdateByCapacity(mockParkingAreaUpdateRequestId,mockParkingAreaUpdateRequest));
+
+        // Verify
+        Mockito.verify(parkingAreaRepository,Mockito.times(1)).findById(mockParkingAreaUpdateRequestId);
+        Mockito.verify(parkingAreaRepository,Mockito.times(0)).save(Mockito.any(ParkingAreaEntity.class));
+
+    }
+
+
+    @Test
+    void givenNullParkingAreaUpdate_whenUpdateParkingArea_thenThrowParkingAreaCanNotBeNull(){
+
+        // Given
+        final ParkingAreaUpdateRequest mockParkingAreaUpdateRequest = new ParkingAreaUpdateRequestBuilder()
+                .withValidField()
+                .withCapacity(null)
+                .build();
+
+        final String mockParkingAreaId = UUID.randomUUID().toString();
+
+        // Then
+        Assertions.assertThrowsExactly(
+                ParkingAreaCapacityCanNotBeNullException.class,
+                () -> parkingAreaUpdateService.parkingAreaUpdateByCapacity(mockParkingAreaId,mockParkingAreaUpdateRequest)
+        );
+
+    }
```

Comment:
Bu teste gerek yok, zaten bu kontrol request nesnesi içerisinde yapılıyor. Böyle bir isteğin servise düşmemesi gerekiyor zaten.